src/main/java/com/spotify/docker/client/gcr/GCloudProcess.java

```diff
@@ -0,0 +1,31 @@
+/*-
+ * -\-\-
+ * docker-client
+ * --
+ * Copyright (C) 2016 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.docker.client.gcr;
+
+import java.io.IOException;
+
+public class GCloudProcess {
+
+  public Process runGcloudDocker() throws IOException {
+    return Runtime.getRuntime().exec("gcloud docker -a");
```

Comment:
I think it might be better to run this like `gcloud --account <account name> docker ...` so that the user can specify which account the credentials come from. Otherwise this is forcing that the "active account" in the user's/host's gcloud installation is the one to use for docker credentials. I think it would be worth decoupling those.